src/applications/vaos/tests/e2e/community-care.e2e.spec.js

```diff
@@ -0,0 +1,126 @@
+const Timeouts = require('../../../../platform/testing/e2e/timeouts.js');
+const VAOSHelpers = require('./vaos-helpers');
+const Auth = require('../../../../platform/testing/e2e/auth');
+
+const continueButton = '.rjsf [type="submit"]';
+
+module.exports = {
+  after: (client, done) => {
+    client.deleteCookies();
+    client.end();
+    done();
+  },
+  login: client => {
+    const token = Auth.getUserToken();
+
+    VAOSHelpers.initAppointmentListMock(token);
+
+    Auth.logIn(
+      token,
+      client,
+      '/health-care/schedule-view-va-appointments/appointments/',
+      3,
+    ).waitForElementVisible('#appointments-list', Timeouts.slow);
+    // .axeCheck('.main');
+  },
+  'Select new appointment': client => {
+    client
+      .click('#new-appointment')
+      .waitForElementVisible(continueButton, Timeouts.normal)
+      .axeCheck('.main');
+  },
+  'Choose the type of care you need': client => {
+    client.click('[value="323"]').perform(() => {
+      client
+        .click('.rjsf [type="submit"]')
+        .pause(Timeouts.slow)
```

Comment:
These timeouts are pretty unreliable, it's better to use one of the `waitFor` commands and look for specific elements on the page.